packages/language-server/src/validation/index.ts

```diff
@@ -31,6 +34,9 @@ export function registerValidationChecks(services: LikeC4Services) {
     IconProperty: iconPropertyRuleChecks(services),
     SpecificationRule: specificationRuleChecks(services),
     Model: modelRuleChecks(services),
+    ModelViews: modelViewsChecks(services),
```

Comment:
I removed this validation as now I see no reason to warn if there are multiple `views` blocks in a document, why not?

What do you think?

Reply:
It looks like a rebase error. I didn't intend to restore it